File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.java
// Last updated: 7/30/2025, 12:47:04 PM
class Solution {
    public int[] runningSum(int[] nums) {
        int[] ans = new int[nums.length];
        int sum = 0;
        for(int i=0; i<nums.length; i++){
            sum += nums[i];
            ans[i] = sum;
        }
        return ans;
    }
}

File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.py
# Last updated: 7/30/2025, 12:23:04 PM
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        # sum = 0
        for i in range(1,len(nums)):
            nums[i]+=nums[i-1]
        return nums